### PR TITLE
Create login/registration page

### PR DESCRIPTION
**Pull request by AnnieDebbie**
This PR addresses #20

### CODE REVIEW

File: newsletters/welcome.html
Comment (anurag629) [inline newsletters/welcome.html:1]: Can you please remove the newsletter from this PR

**Review by anurag629 — COMMENTED** [review]
Great @AnnieDebbie, I'm impressed! 
Left one comment

**Comment by anurag629** [comment]
Unified Login and Registration Page Design #20

**Comment by anurag629** [comment]
@AnnieDebbie Did you complete the Login Registration?

**Comment by anurag629** [comment]
Hello @AnnieDebbie, please remove the newsletter page from this PR, and then it's ready to be merged."

**Comment by AnnieDebbie** [comment]
Just seeing these comments, will do so now

> Hello @AnnieDebbie, please remove the newsletter page from this PR, and then it's ready to be merged."

**Comment by AnnieDebbie** [comment]
@anurag629 You may check

**Review by anurag629 — APPROVED** [review]
lgtm

**Pull request merged**